packages/service-backup_python/packaging

```diff
@@ -6,11 +6,11 @@
 
 set -e
 
-tar zxf sqlite/sqlite-autoconf-3420000.tar.gz
+tar zxf sqlite/sqlite-autoconf-3450000.tar.gz
```

Comment:
can we create a story to have these updates automated in the future?
in the story, please add where you grabbed the latest blobs (the url or other instructions)

thanks

Reply:
story created -> https://jira-pivotal.atlassian.net/browse/REDVM-408